features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Subscription/List.jsx

```diff
@@ -0,0 +1,296 @@
+                                    id='AdminPages.Microgateways.List.help.link.three'
+                                    defaultMessage='View the Microgateway labels'
+                                />
+                            )}
+                            />
+
+                        </Link>
+                    </ListItem>
+                </List>
+            </HelpBase>),
+        pageStyle: 'half',
+        title: intl.formatMessage({
+            id: 'Throttling.Subscription.Policy.search.default',
+            defaultMessage: 'Subscription Throttling Policies',
+        }),
+        EditTitle: intl.formatMessage({
+            id: 'Throttling.Subscription.Policy.search.default',
+            defaultMessage: 'Subscription Throttling Policies',
+        }),
+    };
+
+    const columProps = [
+        {
+            name: 'name',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Subscription.Throttling.policy.table.header.name',
+                defaultMessage: 'Name',
+            }),
+            options: {
+                filter: true,
+                sort: true,
+            },
+        },
+        {
+            name: 'quotaPolicy',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Subscription.Throttling.policy.table.header.quota.policy',
+                defaultMessage: 'Quota Policy',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+        {
+            name: 'quota',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Subscription.Throttling.policy.table.header.quota',
+                defaultMessage: 'Quota',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+        {
+            name: 'unitTime',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Subscription.Throttling.policy.table.header.unit.time',
+                defaultMessage: 'Unit Time',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+        {
+            name: 'rateLimit',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Subscription.Throttling.policy.table.header.rate.limit',
+                defaultMessage: 'Rate Limit',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+        {
+            name: 'timeUnit',
+            label: intl.formatMessage({
+                id: 'Admin.Throttling.Subscription.Throttling.policy.table.header.time.unit',
+                defaultMessage: 'Time Unit',
+            }),
+            options: {
+                filter: true,
+                sort: false,
+            },
+        },
+    ];
+
+    const emptyBoxProps = {
```

Comment:
@chathuranga95 , Did we implement the `emptyBoxProps` alterations we discussed few days ago?